src/components/Rockets.module.css

```diff
@@ -2,14 +2,39 @@
   margin-top: 2rem;
   display: grid;
   grid-template-columns: 1fr 4fr;
+  gap: 2rem;
   padding: 1rem 6.24rem;
 }
 
-.rocketContainer button {
+.images {
+  height: 100%;
+  width: 100%;
+}
+
+.reservation {
   font-size: 1rem;
   padding: 1rem 0.8rem;
   background-color: rgb(0, 123, 255);
   border-radius: 10px;
   color: white;
   border: none;
 }
+
+.cancelReservation {
```

Comment:
class name in this order isn't so famous. How about using the same traditional order?